tests/unit/lib/helpers.js

```diff
@@ -3,7 +3,7 @@ const { mockRequest, mockResponse } = require("jest-mock-req-res");
 const JourneyModel = require("hmpo-form-wizard/lib/journey-model");
 const WizardModel = require("hmpo-form-wizard/lib/wizard-model.js");
 
-global.createDefaultReqResNext = () => {
```

Comment:
So the idea behind all of these `req`/`res`/`next` spies is to be able to put standard objects available for use inside Jest functions. The use of `createDefaultReqResNext` is because destructing unfortunately breaks the object references on req. This is setup for each test through the use of [setupfilesafterenv](https://jestjs.io/docs/configuration#setupfilesafterenv-array)

Occasionally its useful to be able to create a set of Express `req`/`res`/`next` spies, so scoping this out of global removes that ability.

Explicitly scoping these `req`/`res`/`next` either adds noise or clarity, depending upon how you look on it.

Finally, these files are repeated throughout various repos, so a future refactoring would be to move them into something like `@govuk-one-login/jest-test-helpers`

Reply:
We can keep it global - at the moment this method is only used in this setup file, though, and the linter complains.
It wants to prefix global things with `global.` when using them so we can do that instead if we want to (like with the `req/res/next` objects).

Reply:
Or we'd have to manually require the file that sets up the spies which kind of defeats the purpose of the setup files. The linter doesn't know where those global objects come hence it throws an error.

Reply:
OK this can be done with config instead to tell eslint about the globals